DOC-5283: Serialization review (#1507)
corda/corda-docs-portal: content/en/platform/corda/5.0/developing-applications/api/api-serialisation.md

them will automatically be allowed.
{{< /note >}}

{{< note >}}

> Custom serializers targeting Corda platform types or JDK types are not allowed. They will fail to be registered to Corda.

Reviewer: ```suggestion
Custom serializers targeting Corda platform types or JDK types are not allowed. They will not be registered with Corda.
```
or
```suggestion
Custom serializers targeting Corda platform types or JDK types are not allowed. They will fail to register with Corda.
```
or
```suggestion
Custom serializers targeting Corda platform types or JDK types are not allowed. Corda will not register them.
```

### Inaccessible Private Properties

The Corda AMQP serialization framework does not support private properties without publicly

> accessible getter methods. The below example will fail at serialization.

Reviewer: ```suggestion
accessible getter methods. The below example fails at serialization.
```
